subprojects/core/src/main/java/org/gradle/internal/service/scopes/BuildScopeServices.java

```diff
@@ -153,6 +153,7 @@
 import org.gradle.internal.reflect.DirectInstantiator;
 import org.gradle.internal.reflect.Instantiator;
 import org.gradle.internal.resource.TextResourceLoader;
+import org.gradle.groovy.scripts.internal.BuildOperationBackedScriptCompilationHandler;
```

Comment:
:x: import doesn't feel right at that place

Reply:
fixed